app/config/config.neon

```diff
@@ -185,6 +192,7 @@ services:
 			- addOnBuildReady(@CI\Builds\PhpCs\PublishPhpCs)
 			- addOnBuildReady(@CI\Builds\Tests\PublishTests)
 			- addOnBuildReady(@CI\Builds\PhpStan\PublishPhpStan)
+			- addOnBuildReady(@CI\Builds\Cypress\PublishCypress)
```

Comment:
Takto to není vhodné, protože došlo teprve ke composeru, ale ještě není hotové `make build-staging-front`. Je potřeba přidat nový event do vytvoření a aktualizace.

Reply:
Pravda, pravda, přidáno.